app.js

```diff
@@ -60,6 +61,17 @@ function authorizationMiddleware(req, res, next) {
 
 app.use(authorizationMiddleware);
 
+/**
+ * Middleware that adds the account sid (taken from path) to the request object, so we can use it in the routes.
+ * NOTE: If we ever move this project to Typescript: https://dev.to/kwabenberko/extend-express-s-request-object-with-typescript-declaration-merging-1nn5
+ */
+const addAccountSid = (req, res, next) => {
+  req.accountSid = req.params.accountSid;
+  return next();
+};
+
+app.use('/v0/accounts/:accountSid', addAccountSid, apiV0);
+
 // run with node app.js and hit curl localhost:8080/contacts/
 app.get('/contacts', async (req, res) => {
```

Comment:
Your PR looks great! However I've seen that we're keeping the old routes in app.js file. Is this intentional? Should we delete them and start using the new routes?

Reply:
Yes, that's intentional, so we can change the endpoints being used in the frontend without breaking it. I'll remove them when we all feel confident that this is working fine.